mantidproject/mantid
# Muon analysis old has muon fittting funcs

~~Going to use #27818 to move the new muon categories into a overall muon category.~~

**Description of work.**
The muon fitting functions have been moved into a muon category, which has 3 subcategories. 



**To test:**


Open MantidPlot
Open Muon Analysis Old
Load some data
Go to the data analysis tab
Add a fitting function (there should be 3 muon categories inside Muon) 
Fixes #27768. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
